CHANGELOG.md

```diff
@@ -126,6 +126,7 @@
   shortened labels for entries with long module paths. When an option is
   selected from the dropdown, the necessary module imports are inserted,
   eliminating the need for fully qualified names.
+- [Icon buttons now have tooltips.][6035]
```

Comment:
Please write a better description - it is user-facing changelog.

Reply:
Any suggestions? Please be constructive when providing PR comments.

Reply:
I was sure it was constructive enough. User-facing changelog should describe what is the change, why it was made and what benefits it gives users.

Reply:
I played with GPT for a minute:
```
[Added tooltips to icon buttons][6035] for improved usability. Users can now quickly understand each button's function.
```